<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PagePalette_PagesBase</class>
 <widget class="QWidget" name="PagePalette_PagesBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arrange Pages</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="masterPageLabel">
         <property name="text">
          <string>Available Master Pages:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SeList" name="masterPageList">
         <property name="toolTip">
          <string>&lt;qt&gt;List of master pages in the document. Master page names may be dragged onto the page view below to apply master pages, or onto the empty space between pages to create new pages.&lt;/qt&gt;</string>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="pageViewLabel">
         <property name="text">
          <string>Document Pages:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SeView" name="pageView">
         <property name="toolTip">
          <string>&lt;qt&gt;List of normal pages in the document, shown with the document layout. Pages may be dragged to rearrange or delete them.&lt;/qt&gt;</string>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="rowCount">
          <number>1</number>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="PageLayouts" name="pageLayout">
       <property name="title">
        <string>Document Layout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TrashBin" name="trash">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;qt&gt;Drag pages or master pages onto the trash to delete them&lt;/qt&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SeList</class>
   <extends>QListWidget</extends>
   <header>ui/pagepalette_widgets.h</header>
  </customwidget>
  <customwidget>
   <class>SeView</class>
   <extends>QTableWidget</extends>
   <header>ui/pagepalette_widgets.h</header>
  </customwidget>
  <customwidget>
   <class>PageLayouts</class>
   <extends>QGroupBox</extends>
   <header>ui/pagelayout.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrashBin</class>
   <extends>QLabel</extends>
   <header>ui/pagepalette_widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
